.github/workflows/gradle-check.yml

```diff
@@ -0,0 +1,71 @@
+name: Jenkins Gradle Check
+on: [pull_request]
+
+jobs:
+  gradle-check:
+    runs-on: ubuntu-latest
+    timeout-minutes: 130
```

Comment:
As the last PR was closed. 
How are we deciding the timeout here to be 130? Did we document it somewhere?

Reply:
Offline discussion with @owaiskazi19 @dreamer-89 in this url: https://github.com/opensearch-project/opensearch-build/issues/851#issuecomment-1167932678